door43_tools/obs_handler.py

```diff
@@ -29,68 +29,56 @@ def __contains__(self, item):
 
 
 class OBSInspection(object):
-    def __init__(self, content_dir=None):
+    def __init__(self, filename, chapter=None):
```

Comment:
Since the parameters have changed, the doc comments below are no longer correct.